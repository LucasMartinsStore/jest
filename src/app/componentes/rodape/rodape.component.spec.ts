import { ComponentFixture, TestBed } from '@angular/core/testing';
import { RodapeComponent } from './rodape.component';

describe('CabecalhoComponent', () => {
  let component: RodapeComponent;
  let fixture: ComponentFixture<RodapeComponent>;

  beforeEach(() => {
    TestBed.configureTestingModule({
      imports: [RodapeComponent],
    });
    fixture = TestBed.createComponent(RodapeComponent);
    component = fixture.componentInstance;
  });

  it('should create', () => {
    expect(component).toBeTruthy();
  });

  it('should defined property alt and src', () => {
    expect(component.alt).toBeDefined();
    expect(component.src).toBeDefined();
  });

  it('should render the image with the correct alt and src', () => {
    component.src = 'https://example.com/test-image.jpg';
    component.alt = 'Imagem teste';
    expect(component).toMatchSnapshot();
  });
});
