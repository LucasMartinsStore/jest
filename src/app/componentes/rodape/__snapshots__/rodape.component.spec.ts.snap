// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CabecalhoComponent should render the image with the correct alt and src 1`] = `
RodapeComponent {
  "__ngContext__": 4,
  "alt": "Imagem teste",
  "src": "https://example.com/test-image.jpg",
}
`;
